FileUpload/src/FileUpload.jsx

```diff
@@ -67,7 +67,7 @@ export default class FileUpload extends Component {
         }
         if (nextProps.selectedFile.fileId !== this.state.selectedFile.fileId) {
             const file = nextProps.selectedFile;
-            this.updateFileState(file);
+            //this.updateFileState(file);
```

Comment:
This is an issue caused by commit 2696ac04888afa426a5f84a20d25755ff852269f, as DNN-10146 still in progress, i just comment this line, and the function should be handled by DNN-10146, please add comment on that JIRA after this merged. thx